backend/app/routers/ontologies.py

```diff
@@ -0,0 +1,23 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm import Session
+
+from ..db import get_db
+from ..core.ipfs import ipfs_client
+from ..schemas import DigitalTwinInterface
+from ..models import get_or_create_ontology
+
+
+router = APIRouter(
+    prefix="/ontology",
+    tags=["ontologies"],
+)
+
+
+@router.post("/")
+async def save_ontology(ontology: DigitalTwinInterface, db: Session = Depends(get_db)):
+    ontology_hash = ipfs_client.save_json(ontology.dict())
```

Comment:
The hash needs still to be defined right? Because this is the dict

Reply:
@amecklenborg the hash is returned by IPFS. It is the unique identifier of record saved.